lib/main/STM32F7/Middlewares/ST/STM32_USB_Device_Library/Class/HID/Src/usbd_hid.c

```diff
@@ -317,26 +317,34 @@ __ALIGN_BEGIN static uint8_t USBD_HID_DeviceQualifierDesc[USB_LEN_DEV_QUALIFIER_
 
 __ALIGN_BEGIN static uint8_t HID_MOUSE_ReportDesc[HID_MOUSE_REPORT_DESC_SIZE]  __ALIGN_END =
 {
-        0x05, 0x01,                    // USAGE_PAGE (Generic Desktop)
-        0x09, 0x05,                    // USAGE (Game Pad)
-        0xa1, 0x01,                    // COLLECTION (Application)
-        0xa1, 0x00,                    //   COLLECTION (Physical)
-        0x05, 0x01,                    //     USAGE_PAGE (Generic Desktop)
-        0x09, 0x30,                    //     USAGE (X)
-        0x09, 0x31,                    //     USAGE (Y)
-        0x09, 0x32,                    //     USAGE (Z)
-        0x09, 0x33,                    //     USAGE (Rx)
-        0x09, 0x35,                    //     USAGE (Rz)
-        0x09, 0x34,                    //     USAGE (Ry)
-        0x09, 0x40,                    //     USAGE (Vx)
-        0x09, 0x38,                    //     USAGE (Wheel)
-        0x15, 0x81,                    //     LOGICAL_MINIMUM (-127)
-        0x25, 0x7f,                    //     LOGICAL_MAXIMUM (127)
-        0x75, 0x08,                    //     REPORT_SIZE (8)
-        0x95, 0x08,                    //     REPORT_COUNT (8)
-        0x81, 0x02,                    //     INPUT (Data,Var,Abs)
-        0xc0,                          //   END_COLLECTION
-        0xc0                           /*     END_COLLECTION                 */
+  0x05, 0x01,                    // USAGE_PAGE (Generic Desktop)
```

Comment:
All these changes under `lib/` are changes to code imported from STM - they will have to be grafted whenever the STM libraries are updated (unless STM fixes their USB HID implementation).
To make this easier, can you please:
- extract these changes into a separate pull request;
- revert the part of the changes that are whitespace / formatting changes only, so that only code changes remain in `lib/`.
This will make future handling of these changes much easier.

Reply:
No probs at all.
I'll revert those whitespaces / formattings and will made another PR with those files in `lib/` only.
And just to be clear: changes in `lib/` within this PR: should I leave them as is, or remove it from this PR completely?
Thanks in advance.

Reply:
@mikeller , No probs at all.
I'll revert those whitespaces / formattings and will made another PR with those files in lib/ only.
And just to be clear: changes in lib/ within this PR: should I leave them as is, or remove it from this PR completely?
Thanks in advance.

Reply:
@mikeller , No probs at all.
I'll revert those whitespaces / formattings and will make another PR with those files in `lib/` only.
And just to be clear: changes in `lib/` within this PR: should I leave them as is, or remove it from this PR completely?
Thanks in advance.

Reply:
@mikeller , no probs at all. Just done that.